schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/config/associated-panel/default.json

```diff
@@ -439,6 +439,38 @@
           }
         }
       },
+      {
+        "group": "onlineDownload",
+        "label": "onlineDownloadOGCFeatures",
+        "copyLabel": "desc",
+        "icon": "fa gn-icon-onlinesrc",
+        "sources": {
+          "metadataStore": {
+            "label": "searchAservice",
+            "params": {
+              "serviceType": ["OGC API - Features"]
+            }
+          }
+        },
+        "process": "onlinesrc-add",
+        "fields": {
+          "url": {
+            "isMultilingual": false
+          },
+          "protocol": {
+            "value": "OGC API - Features",
```

Comment:
I have thought again about this value here; I think it would at least be better to align with the "official" values that should land in GeoNetwork soon, see https://github.com/geonetwork/core-geonetwork/pull/7468

The value should then be `OGC API Features` instead of `OGC API - Features`. Otherwise records in geOrchestra might not be correctly understood once the PR above is merged.

Sorry for not picking that up earlier :confused:

Reply:
Thanks @jahow !
We'll take care of updating this locally.

Reply:
cc @f-necas